modules/core/03-connection/types/expected_keepers.go

```diff
@@ -16,9 +16,10 @@ type ClientKeeper interface {
 	GetClientConsensusState(ctx sdk.Context, clientID string, height exported.Height) (exported.ConsensusState, bool)
 	GetSelfConsensusState(ctx sdk.Context, height exported.Height) (exported.ConsensusState, error)
 	ValidateSelfClient(ctx sdk.Context, clientState exported.ClientState) error
+	VerifyMembershipp(ctx sdk.Context, clientID string, height exported.Height, delayTimePeriod uint64, delayBlockPeriod uint64, proof []byte, path exported.Path, value []byte) error
```

Comment:
I had to be "creative" with the name of the function here because there is already a [`VerifyMembership` function implemented in the keeper](https://github.com/cosmos/ibc-go/blob/356e433cf99e9d69d718f4fd46e2f85a8d197743/modules/core/02-client/keeper/grpc_query.go#L328)... Welcome to ideas on how to solve this!

Reply:
`VerifyMembershipProof` and `VerifyNonMembershipProof`?

Reply:
`VerifyExistence` and `VerifyNonExistence`?